armi/reactor/components/complexShapes.py

```diff
@@ -185,6 +185,13 @@ class Helix(ShapedComponent):
     http://mathworld.wolfram.com/Helix.html
     In a single rotation with an axial climb of P, the length of the helix will be a factor of
     2*pi*sqrt(r^2+c^2)/2*pi*c longer than vertical length L. P = 2*pi*c.
+
+    - od: outer diameter of the helix wire
+    - id: inner diameter of the helix wire (if non-zero, helix wire is annular.)
+    - axialPitch: vertical distance between wraps
```

Comment:
call out "full revolution" , and mention 2pi or 360 degrees to be explicit

Reply:
Addressed. https://github.com/terrapower/armi/pull/747/commits/f03f374c117b2fece94b2b1beec752bd7c1a103d